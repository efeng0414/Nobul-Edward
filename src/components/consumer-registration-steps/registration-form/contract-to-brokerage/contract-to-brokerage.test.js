import React from "react";
import ContractToBrokerageFormItem from "./index";

describe("ContractToBrokerageFormItem", () => {
  const mockForm = {
    getFieldDecorator: () => jsx => jsx
  };
  const componentProps = {
    intl: global.getIntl(),
    fieldName: "Example field",
    getFieldDecorator: mockForm.getFieldDecorator
  };

  const componentJSX = <ContractToBrokerageFormItem {...componentProps} />;

  describe("Component Snapshot", () => {
    const componentSnapshot = global.renderer.create(componentJSX).toJSON();
    it("should match stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
