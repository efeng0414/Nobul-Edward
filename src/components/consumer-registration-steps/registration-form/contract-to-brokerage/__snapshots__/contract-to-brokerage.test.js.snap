// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContractToBrokerageFormItem Component Snapshot should match stored snapshot 1`] = `
<div>
  <div
    className="consumer-registration-form-item contract-brokerage"
  >
    <div
      className="ant-row ant-form-item"
    >
      <div
        className="ant-form-item-control-wrapper"
      >
        <div
          className="ant-form-item-control"
        >
          <span
            className="ant-form-item-children"
          >
            <p
              className="consumer-registration-paragraph"
            >
              Are you currently under contract to a brokerage?
            </p>
            <div
              className="ant-radio-group ant-radio-group-outline"
            >
              <label
                className="ant-radio-wrapper"
              >
                <span
                  className="ant-radio"
                  style={Object {}}
                >
                  <input
                    checked={false}
                    className="ant-radio-input"
                    disabled={false}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    type="radio"
                    value={false}
                  />
                  <span
                    className="ant-radio-inner"
                  />
                </span>
                <span>
                  No
                </span>
              </label>
              <label
                className="ant-radio-wrapper"
              >
                <span
                  className="ant-radio"
                  style={Object {}}
                >
                  <input
                    checked={false}
                    className="ant-radio-input"
                    disabled={false}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    type="radio"
                    value={true}
                  />
                  <span
                    className="ant-radio-inner"
                  />
                </span>
                <span>
                  Yes
                </span>
              </label>
            </div>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
